.github/workflows/build-docs.yml

```diff
@@ -3,73 +3,40 @@ name: Docs
 on:
   push:
     branches:
-      - stable
+      - rename/aicspylibczi
+    tags:
+      - 'docs'
```

Comment:
Change this to master without the tags before merging.

Reply:
done for whatever reason I can't get tags to work at all.